src/move/econia/sources/user.move

```diff
@@ -1772,12 +1773,14 @@ module econia::user {
               &mut market_account_ref_mut.quote_available,
               base_fill, quote_fill);
         // Assert no inbound asset overflow.
-        assert!((in_fill + (*in_ceiling_ref_mut as u128)) <= (HI_64 as u128),
-                E_OVERFLOW_ASSET_IN);
+        if (in_fill + (*in_ceiling_ref_mut as u128) > (HI_64 as u128)) {
+            in_fill = (HI_64 as u128) - (*in_ceiling_ref_mut as u128);
+        };
         // Assert enough outbound asset to cover the fill, which also
         // ensures outbound fill amount does not overflow.
-        assert!((out_fill <= (*out_available_ref_mut as u128)),
-                E_NOT_ENOUGH_ASSET_OUT);
+        if (out_fill > (*out_available_ref_mut as u128)) {
+            out_fill = (*out_available_ref_mut as u128);
+        };
```

Comment:
Same here on the silent reassingment, this can create an accounting error against the order state on the order book.